src/lib/viewers/image/ImageLoader.js

```diff
@@ -65,6 +72,31 @@ class ImageLoader extends AssetLoader {
             return viewer.REP === entry.representation;
         });
     }
+
+    /**
+     * @inheritdoc
+     */
+    determineViewer(file, disabledViewers = []) {
+        return this.viewers.find(viewer => {
+            if (disabledViewers.indexOf(viewer.NAME) > -1 || viewer.EXT.indexOf(file.extension) === -1) {
+                return false;
+            }
+
+            const multiPageViewer = (viewer.ASSET || '').indexOf('{page}') !== -1;
+
+            return file.representations.entries.some(entry => {
+                if (viewer.REP !== entry.representation) {
+                    return false;
+                }
+
+                // If the viewer uses paged assets (e.g., image_2048_jpg), then
+                // ensure that there is a compatible paged representation.
+                const pagedRep = (entry.properties || {}).paged === 'true';
```

Comment:
I'm finding the logic here hard to follow. Can we have non-paged TIFF files? What case is this guarding against?

Reply:
@jstoffan I was seeing the following issue... Suppose we have the following reps available:

```
representations: {                                              
  entries: [                                                  
    {                                                       
      representation: 'jpg',                              
      properties: {                                       
        dimensions: '1024x1024',                        
        paged: 'false',                                 
      },                                                  
    },                                                      
    {                                                       
      representation: 'png',                              
      properties: {                                       
        dimensions: '2048x2048',                        
        paged: 'true',
      },                                                  
    },                                                      
  ],                                                          
}
```

When `AssetLoader.determineViewer(file)` is called for a TIFF file, it will see that a "jpg" representation is available (the first entry), and select the following viewer:

```
{                                                                           
  NAME: 'MultiImage',                                                     
  CONSTRUCTOR: MultiImageViewer,                                          
  REP: 'jpg',                                                             
  EXT: ['tif', 'tiff'],                                                   
  ASSET: '{page}.jpg',                                                    
}
```

However, the 1024x1024 jpg rep that is available is not a paged rep. So, when we then call `ImageLoader.determineRepresentation()`, we get back `null/undefined` because it specifically filters out these reps:

```
determineRepresentation(file, viewer) {                                     
  return file.representations.entries.find(entry => {                     
    // Do not use the dimensions=1024x1024&paged=false rep that is for document preloading
    if (entry.properties && entry.properties.paged === 'false') {       
      return false;                                                   
    }                                                                   
                                                                                
    return viewer.REP === entry.representation;                         
   });                                                                     
}
```

This most recent change aims to only use the `MultiImage` viewer (which is currently only used for TIFF files) when a paged representation exists.